serpentTools/parsers/microxs.py

```diff
@@ -134,22 +132,14 @@ def _storeMicroXS(self, chunk):
             if '%' in tline:
                 tline = tline[:tline.index('%')]
             if len(tline.split()) > 3:
-                values = self._str2num(tline)
+                values = str2vec(tline)
                 # isotope, reaction type and isomeric state
                 reactionData = (int(values[0]), int(values[1]), int(values[2]))
                 currXS[reactionData], currUnc[reactionData] = \
-                    array(splitItems(values[3:]))
+                    array(splitValsUncs(values[3:]))
```

Comment:
No need to pass this through `array`, `splitValsUncs` returns two arrays

Reply:
Changed